<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppInfoView</class>
 <widget class="QMainWindow" name="AppInfoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_topBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(38, 36, 36);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_topBarApp">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_appName">
            <property name="styleSheet">
             <string notr="true">color: rgb(246, 245, 244);
font: 700 13pt &quot;Ubuntu&quot;;</string>
            </property>
            <property name="text">
             <string>AppName v1.0.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>467</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame_topBarWindowControl">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_minimize">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Minimize</string>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-image: url(:/icons/Assets/icons/icon_minimize.png);
background-color: rgb(60, 60, 60);
border:none;
border-radius:15px;
background-repeat: no-repeat;
background-position: center;
}

QPushButton::hover{
background-color: rgb(200, 200, 200);
border:none;
background-repeat: no-repeat;
background-position: center;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_maximize">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Maximize</string>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-image: url(:/icons/Assets/icons/icon_maximize.png);
background-color: rgb(60, 60, 60);
border:none;
background-repeat: no-repeat;
background-position: center;
border-radius:15px;
}

QPushButton::hover{
background-color: rgb(200, 200, 200);
border:none;
background-repeat: no-repeat;
background-position: center;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_exit">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Exit</string>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-image: url(:/icons/Assets/icons/cil-x.png);
background-color: rgb(60, 60, 60);
border:none;
background-repeat: no-repeat;
background-position: center;
border-radius:15px;
}

QPushButton::hover{
background-color: rgb(170, 0, 0);
border-radius:15px;
border:none;
background-repeat: no-repeat;
background-position: center;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(100, 130, 0);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(61, 59, 59);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(39, 39, 39);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="styleSheet">
             <string notr="true">QScrollArea{
background-repeat: no-repeat;
background-position: center;
border:none;

}

 QScrollBar:horizontal
 {

     height: 15px;

     margin: 3px 15px 3px 15px;

     border: 1px transparent #2A2929;

     border-radius: 4px;


 }



 QScrollBar::handle:horizontal

 {

     background-color: white;      /* #605F5F; */

     min-width: 5px;

     border-radius: 4px;

 }



 QScrollBar::add-line:horizontal

 {

     margin: 0px 3px 0px 3px;

     border-image: url(:/qss_icons/rc/right_arrow_disabled.png);

     width: 10px;

     height: 10px;

     subcontrol-position: right;

     subcontrol-origin: margin;

 }



 QScrollBar::sub-line:horizontal

 {

     margin: 0px 3px 0px 3px;

     border-image: url(:/qss_icons/rc/left_arrow_disabled.png);

     height: 10px;

     width: 10px;

     subcontrol-position: left;

     subcontrol-origin: margin;

 }



 QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on

 {

     border-image: url(:/qss_icons/rc/right_arrow.png);

     height: 10px;

     width: 10px;

     subcontrol-position: right;

     subcontrol-origin: margin;

 }





 QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on

 {

     border-image: url(:/qss_icons/rc/left_arrow.png);

     height: 10px;

     width: 10px;

     subcontrol-position: left;

     subcontrol-origin: margin;

 }



 QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal

 {

     background: none;

 }





 QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal

 {

     background: none;

 }



 QScrollBar:vertical

 {

     

	background-color: rgb(49, 51, 50);

     width: 15px;

     margin: 15px 3px 15px 3px;

     border: 1px transparent #2A2929;

     border-radius: 4px;

 }



 QScrollBar::handle:vertical

 {

     background-color: white;         /* #605F5F; */

     min-height: 5px;

     border-radius: 4px;

 }



 QScrollBar::sub-line:vertical

 {

     margin: 3px 0px 3px 0px;

     border-image: url(:/qss_icons/rc/up_arrow_disabled.png);

     height: 10px;

     width: 10px;

     subcontrol-position: top;

     subcontrol-origin: margin;

 }



 QScrollBar::add-line:vertical

 {

     margin: 3px 0px 3px 0px;

     border-image: url(:/qss_icons/rc/down_arrow_disabled.png);

     height: 10px;

     width: 10px;

     subcontrol-position: bottom;

     subcontrol-origin: margin;

 }



 QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on

 {



     border-image: url(:/qss_icons/rc/up_arrow.png);

     height: 10px;

     width: 10px;

     subcontrol-position: top;

     subcontrol-origin: margin;

 }





 QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on

 {

     border-image: url(:/qss_icons/rc/down_arrow.png);

     height: 10px;

     width: 10px;

     subcontrol-position: bottom;

     subcontrol-origin: margin;

 }



 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical

 {

     background: none;

 }





 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical

 {

     background: none;

 }</string>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>407</width>
               <height>381</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(100, 130, 0);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_bottomBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(38, 36, 36);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_size_grip">
       <property name="geometry">
        <rect>
         <x>297</x>
         <y>12</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>SizeFDiagCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QSizeGrip {
	background-image: url(:/icons/Assets/icons/cil-size-grip.png);
	background-position: center;
	background-repeat: no-reperat;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
